box-share-sdk/src/main/java/com/box/androidsdk/share/sharerepo/ShareRepo.java

```diff
@@ -165,11 +165,7 @@ else if (boxCollaborationItem instanceof BoxFolder) {
      * @throws Exception
      */
     public void setExpiryDate(BoxCollaborationItem boxCollaborationItem, Date date) throws Exception {
-        try {
-            handleTaskAndPostValue(mController.executeRequest(BoxItem.class, mController.getCreatedSharedLinkRequest(boxCollaborationItem).setUnsharedAt(date)), mSharedLinkedItem);
-        } catch (Exception e){
-            throw e;
-        }
+        handleTaskAndPostValue(mController.executeRequest(BoxItem.class, mController.getCreatedSharedLinkRequest(boxCollaborationItem).setUnsharedAt(date)), mSharedLinkedItem);
```

Comment:
The comment for this method still states that it throws an exception. Please adjust comment to new method behaviors.

Reply:
the setExpiryDate() section can cause exception. I am propagating the exception upward.

Reply:
See the method header comments, it still says "@throws Exception"

Reply:
I thought the exception will be propagated because of that.